src/main/java/io/micronaut/live/Application.java

```diff
@@ -31,11 +30,7 @@ public void configure(ApplicationContextBuilder context) {
         }
     }
 
-
     public static void main(String[] args) {
-        Micronaut.build(args)
-                .mainClass(Application.class)
-                .defaultEnvironments(DEFAULT_ENV)
-                .start();
+        Micronaut.run(Application.class);
```

Comment:
I think we should keep this block to allow users to run the app in dev environment with `run` and `optimizedRun`.

Reply:
This is handled by the application context customizer already, this is redundant.

Reply:
Application Context customizers for for `run` and `optimizedRun`?

Reply:
Yes, they should eventually become the idiomatic way to customize the application context before running (because it also works for functions, for ex).